src/v2/Apps/Search/Components/GenericSearchResultItem.tsx

```diff
@@ -54,52 +45,31 @@ export class GenericSearchResultItem extends React.Component<
     }
 
     return (
-      <Link
-        href={href}
-        underlineBehavior="none"
-        onClick={() => {
-          this.handleClick()
-        }}
-      >
-        <ItemRow>
-          <Spacer pb={3} />
-          <Flex pl={4} flexDirection="row">
-            <Box height={70} width={70} mr={2}>
-              <Flex
-                style={{ backgroundColor: color("black10") }}
-                height="100%"
-                justifyContent="center"
-                alignItems="center"
-              >
-                {imageUrl && entityType !== "City" ? (
+      <Link href={href} underlineBehavior="none" onClick={this.handleClick}>
+        <ItemRow py={2}>
+          <Flex px={[0, 4]} flexDirection="row">
+            <Box width={72} height={72} mr={2}>
+              <BorderBox width={72} height={72} p={0}>
+                {imageUrl && entityType !== "City" && (
                   <Image width={70} height={70} src={imageUrl} alt={name} />
-                ) : (
-                  <FallbackIcon entityType={entityType} />
                 )}
-              </Flex>
+              </BorderBox>
             </Box>
 
             <Box>
-              <Sans color="black100" size="2" weight="medium">
+              <Text variant="xs" color="black60" mb={0.5}>
                 {translateEntityType(entityType)}
-              </Sans>
-              <Spacer mb={0.5} />
+              </Text>
 
-              <Serif color="black100" size="3">
-                {name}
-              </Serif>
+              <Text size="md">{name}</Text>
 
               {description && (
-                <>
-                  <Spacer mb={0.5} />
-                  <Serif color="black60" size="3" maxWidth={536}>
-                    <Truncator maxLineCount={3}>{description}</Truncator>
-                  </Serif>
-                </>
+                <Text mt={0.5} variant="sm" color="black60">
```

Comment:
This component has a `lineClamp` prop — so you can get rid of that truncator below